spec/salestation/app/errors/dependency_currently_unavailable_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+describe Salestation::App::Errors::DependencyCurrentlyUnavailable do
+  subject(:create_dependency_currently_unavailable) { described_class.new(message: message) }
+
+  let(:message) { 'message' }
+
+  it 'has message' do
+    expect(create_dependency_currently_unavailable.message).to eq(message)
```

Comment:
The setup for this test feels a bit forced, I see where you are going with this, but perhaps just
`expect(described_class.new(message: message).message).to eq(message)`